src/pyhf/modifiers/staterror.py

```diff
@@ -145,18 +151,6 @@ def __init__(self, modifiers, pdfconfig, builder_data, batch_size=None):
             [[builder_data[m][s]['data']['mask']] for s in pdfconfig.samples]
             for m in keys
         ]
-        self.__staterror_uncrt = default_backend.astensor(
```

Comment:
I'm sure this will be obvious after I've slept, but why is this chunk removed?

Reply:
it's not used by anything.

Reply:
Okay so just cleaning of vestigial code.